index.html

```diff
@@ -1388,16 +1388,17 @@ <h2>
       </pre>
 
       <p class="note" title="Determining the DID controller">
-If the <code><a>controller</a></code> property is not present, the
-<a>DID controller</a> can be determined by consulting the respective
-<a>DID method</a> specification. It cannot be assumed that the
-<a>DID subject</a> is also the controller of its own <a>DID</a>, nor
-can it be assumed that <em>only</em> entities identified by <a>DIDs</a>
-present in the value of <code><a>controller</a></code> are capable of exerting
-control over the <a>DID</a> in question. Similarly, <a>DID methods</a> are not
-obliged to respect the value of <code><a>controller</a></code> and can specify
-means for updating the <a>DID document</a> that are unrelated to the
-<code><a>controller</a></code> property.
+If the <code><a>controller</a></code> property is present, its value expresses
+the root of authority for information expressed in the <a>DID Document.</a> If
+the <code><a>controller</a></code> property is not present, the value can be
+inferred to be the same as that of the <code><a>id</a></code> property of the
+<a>DID document</a>. The way to determine how the <a>DID Document</a> can be
+changed is by consulting the relevant <a>DID method</a>. If no <a>verification
+methods</a> are present in the <a>DID Document</a>, then a
+<code><a>controller</a></code> property MUST be present, otherwise the
+the <a>DID document</a> can no longer be changed beyond whatever its present
+value is.
```

Comment:
I'm not comfortable with the proposed text, would this re-wording be acceptable?

```suggestion
If the <code><a>controller</a></code> property is present, its value expresses
the root of authority for information expressed in the <a>DID Document.</a> If
the <code><a>controller</a></code> property is not present, the root of
authority for information expressed in the <a>DID Document</a> is determined
by the applicable <a>DID Method</a> specification. 
```

Reply:
@brentzundel -- How does one write DID-method agnostic verification software if we can't assume that the controller is the subject when `controller` is not present? It's one thing to say that it might not be clear who can further makes changes to the document (or not) when there's no controller and no verification methods present, but that's a matter left to DID methods. The problem, I believe, with your suggestion is that it affects DID Document consumers like DID-method agnostic verification software.

Reply:
If there is no `controller` property and no verification methods, what should DID-method agnostic software assume about the DID Document?

Reply:
@brentzundel, that the root of authority is the DID subject. Unless the software is for updating the DID Document, it doesn't care about whether or not the DID Document can be further changed, it just cares about what is presently in the DID Document.

Reply:
That makes sense, I have adjusted my suggestion accordingly.
I think it is cleaner if this note doesn't also attempt to address  whether or not a DID Document can be further changed.

Reply:
@brentzundel thanks, I'm ok with this change. @jandrieu what do you think?

Reply:
But the root of authority is the DID Controller, which is not the property _controller_, but rather the party(ies) that actually control the DID Document, which is up to the DID Method to specify.

This latest language would require DID Methods to specify a controller field that actually controls the DID Document whenever the actual Controller is not the Subject. However, the DID Method can't actually know whether or not the Controller is the Subject. (And currently methods don't have to rely on the controller property; they are all free to design & implement their own approach to decentralized root of authority).

I would go the other way:
> If the <code>controller</code> property is not present, the root of
authority for information expressed in the <code>DID Document</code> is any entity capable of 
updating the DID Document as specified by the <code>DID Method</code>.

Reply:
Or...
> If the controller property is not present, then root of
authority for information expressed in the DID Document 
cannot be determined for the purpose of verification methods.
Any DID Method that wishes to support verification methods MUST 
provide a controller value. However, DID Methods 
are *not* required to support verification methods, nor are they 
required to rely on verification methods for DID Document updates.

Reply:
@jandrieu -- please see my tweak to your above below. Let me know what you think.

Reply:
changed my suggestion to reflect @jandrieu 's input.

Reply:
Brent. I'm not seeing your suggestion. Is it 
> If no <a>verification
methods</a> are present in the <a>DID Document</a>, then a
<code><a>controller</a></code> property MUST be present, otherwise the
the <a>DID document</a> can no longer be changed beyond whatever its present

If I'm tracking the "current" proposal, then this is equivalent to adding a requirement that the DID Method actually respect the "controller" value AND must put EITHER verification methods or a controller property.

The fact is, today, a DID Method 
1. *can* ignore the value of a controller property
2. *can* leave controller property empty even when there are no verification methods
3. *can* provide a DID Document update operation that uses neither verification methods nor the controller value

What I'm hearing is that #3 is something that would like to be changed: that DID Methods MUST provide a DID Update mechanism that uses either Verification Methods or the controller property.  This would be a new requirement.

Reply:
the text I've recommended is here: https://github.com/w3c/did-core/pull/371#discussion_r469486922

I don't think #3 should be changed, or if it is, it should explicitly allow a DID Method to include neither a controller nor a verification property.

Reply:
Hmmm... I need to talk more offline with Dave, I think. The current language (your latest per that comment) suffers from a lack of clarity of what "root authority" means.

I was focused on the fact that the ultimate authority for the content of a DID Document is the DID Controller as supported by the DID Method. This Controller is the party that ACTUALLY can change the document.  The "controller" property is NOT about the ultimate authority as it may or may not indicate the Controller per the DID Method.

So when the "controller" property is called the "root authority", my initial read is that is meant to clarify who has the lowest level control over a DID Document: who can update or create the document.

But what Dave means is about verification methods and the authority for using such methods.

Conflating the two notions of Controller and "controller" AND the two notions of "root" and "ultimate" authority is what has this text wrapped around the axle.

I think we should change the property from "controller" to something else. The namespace collision with "Controller" is a nightmare that is only going to continue as more developers start working with DIDs.

Secondly, I think we should define and use different terms for the "root authority" for verification methods and the ultimate authority to update the document.

Reply:
@jandrieu @dlongley [The suggestion above](https://github.com/w3c/did-core/pull/371/files#r469486922) contains the text we all agreed to during our call today.

@rhiaro please let us know if [this text](https://github.com/w3c/did-core/pull/371/files#r469486922) is also acceptable to you.

Reply:
+1 to the above text. Thanks!

Reply:
Thanks for working through this all! My only quibble is the normative language, as this is a note. I could just lowercase the SHOULD when I update the PR, or we could alter it to not say may and should at all - any preference?

Reply:
Could we change it from a note to a regular paragraph?

Reply:
+1 to the above text AND to change it from a not to a regular paragraph.

Reply:
+1 to the text and to change it from a note to a regular paragraph.